daffodil-propgen/src/main/resources/org/apache/daffodil/xsd/dfdlx.xsd

```diff
@@ -144,4 +145,16 @@
     </xs:restriction>
   </xs:simpleType>
 
+  <xs:attributeGroup name="RuntimePropertiesAG">
+    <xs:attribute form="qualified" name="runtimeProperties" type="dfdlx:RuntimePropertiesType" />
+  </xs:attributeGroup>
+
+  <xs:attributeGroup name="RuntimePropertiesAGQualified">
+    <xs:attribute form="qualified" name="runtimeProperties" type="dfdlx:RuntimePropertiesType" />
+  </xs:attributeGroup>
+
+  <xs:simpleType name="RuntimePropertiesType">
+    <xs:restriction base="xs:string" />
```

Comment:
To follow up on this; however, a list type can be separated by any whitespace, your parsing code looks for single spaces. Only attributes get their values normalized to single spaces. While infrequently used, we do have this Element-form syntax like:
```
<dfdl:property name="dfdlx:runtimeProperties">key=value1
key=value2
key=value3
</dfdl:property>
```
And in that situation the list would not be separated by individual spaces, but by whitespace generally, possibly more than 1 character. 
You can park this as a separate JIRA ticket that dfdlx:runtimeProperties is incompatible with dfdl:property style annotations.

Reply:
I think I can just change the split to happen on `\s+` instead of single-space to support that. I'd rather add that now while this is all fresh in my mind.